php/class-plugin.php

```diff
@@ -36,7 +36,6 @@ public function __construct() {
 	 */
 	public function init() {
 		$this->config = apply_filters( 'customize_pane_resizer_plugin_config', $this->config, $this );
-		$this->min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
```

Comment:
As an eventual core candidate, I think it will be easier to iterate/callaborate without having to worry about minnification, considering this would eventually be rolled into core and compiled with core build scripts. If you don't like this approach, we can change this, but I'm not sure there are enough benefits to warrant minification of an admin script for a feature plugin. Let me know.